content/src/service/Service.ts

```diff
@@ -32,20 +32,20 @@ export interface MetaverseContentService {
     auditInfo: LocalDeploymentAuditInfo,
     origin: string,
     repository?: RepositoryTask | Repository
-  ): Promise<Timestamp>
+  ): Promise<Timestamp | ErrorList>
```

Comment:
We should probably create a type for `Timestamp | ErrorList`. Something like `DeploymentResult` maybe?